pkg/pool-stable/contracts/ComposableStablePoolFactory.sol

```diff
@@ -23,13 +25,28 @@ import "@balancer-labs/v2-pool-utils/contracts/factories/FactoryWidePauseWindow.
 
 import "./ComposableStablePool.sol";
 
-contract ComposableStablePoolFactory is BasePoolSplitCodeFactory, FactoryWidePauseWindow {
+contract ComposableStablePoolFactory is IVersion, IPoolVersion, BasePoolSplitCodeFactory, FactoryWidePauseWindow {
```

Comment:
`BasePoolSplitCodeFactory` or `BasePoolFactory`?

Reply:
`BasePoolSplitCodeFactory` was eventually renamed to `BasePoolFactory`, which was deleted.

Reply:
I'm not sure on the question here. Why would we switch from the factory type that deployed the previous pool?

Reply:
I just wanted to double check if there was any difference between the two (`BasePoolSplitCodeFactory` or `BasePoolFactory`).

`BasePoolSplitCodeFactory` was present in the base, and `BasePoolFactory` is in master. I wasn't sure whether this was a change that we wanted to introduce as part of the changes between composable v1 and composable v2 or not, but if it's just a name change it should be the same.